sonar-plugin-api/src/main/java/org/sonar/api/utils/XpathParser.java

```diff
@@ -56,7 +56,9 @@
  * XML Parsing tool using XPATH. It's recommended to use StaxParser when parsing big XML files.
  *
  * @since 1.10
+ * @deprecated since 5.6 plugins should use their own dependencies
  */
+@Deprecated
 public class XpathParser {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)